spec/lib/sirius_api/people_authorizer_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'sirius_api/people_authorizer'
+require 'sirius_api/user'
+require 'sirius_api/scopes'
+
+describe SiriusApi::PeopleAuthorizer do
+
+  let(:scope) { SiriusApi::Scopes::READ_PERSONAL }
+  let(:user) { SiriusApi::User.new('ALICE', scope) }
+  subject(:authorizer) { described_class.new(user) }
+
+  describe '#authorize_request!' do
+
+    context 'for READ_PERSONAL scope' do
+      it 'allows request to personal endpoint' do
+        expect { authorizer.authorize_request!(:get, '/people/:username', {username: 'ALICE'}) }.not_to raise_error
+      end
+
+      it 'denies request to endpoint for other user' do
+        expect { authorizer.authorize_request!(:get, '/people/:username', {username: 'BOB'}) }.to raise_error(SiriusApi::Errors::Authorization)
```

Comment:
This line is too long (144 chars). Please keep it under 100 chars. You can improve it like this:

```rb
expect {
  authorizer.authorize_request!(:get, '/people/:username', {username: 'BOB'})
}.to raise_error(SiriusApi::Errors::Authorization)
```